lib/widget/input/form_checkbox.dart

```diff
@@ -149,19 +162,53 @@ class OnOffState extends FormFieldWidgetState<OnOffWidget> {
   /// we adjust the hit area of the checkbox here. 40px is smaller than the default (48px)
   /// but it seem to be reasonable for touch device, plus the alignment inside
   /// form is much better (align well with the rest of the input widgets)
+
   Widget get aCheckbox {
     return SizedBox(
         width: 40,
         height: 40,
         child: Checkbox(
+            side: widget._controller.inactiveColor != null
```

Comment:
Why are we decorating the border if it's inActive? The user sets inActive thinking it's equivalent to !active, but instead the fillColor stays the same but they get a different border? This doesn't make sense.

Reply:
Can you read the above description of the PR? I just updated